README.md

```diff
@@ -31,6 +31,31 @@ The Commerce App is a sample application that demonstrates the use of ListFeed p
 
 [Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/Commerce)
 
+### SimpleCalculator App
+
+The SimpleCalculator App is a sample application that is designed to perform basic arithmetic operations. The app was built using the four variants of the [Simple Calc workshop](https://aka.platform.uno/simplecalc-workshop), combining markup language (XAML or C# Markup) and presentation framework (MVVM or MVUX).
+
+[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/SimpleCalc)
+
+### Counter App
+
+Discover the simplicity and power of Uno.Extensions with the Counter App, a straightforward yet powerful demonstration of basic and advanced features of the Uno Platform. This app provides a hands-on experience with fundamental concepts such as state management, user interaction, and UI updates in real-time.
+This sample app was built using the four variants of the [Counter workshop](https://aka.platform.uno/counter-tutorial), combining markup language (XAML or C# Markup) and presentation framework (MVVM or MVUX).
+
+[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/Counter)
+
+### ToDo App
+
+Dive into the essentials of task management with the ToDo App, a meticulously designed sample application that showcases the power and flexibility of Uno.Extensions. By focusing on creating and managing a to-do list, this app demonstrates practical applications of core Uno.Extensions features, offering a hands-on experience in developing responsive, user-friendly interfaces across platforms.
+
+[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/ToDo)
+
+### TubePlayer App
+
+The TubePlayer App is a sample application that leverages Uno Platform to recreate the core functionalities of YouTube. This app serves as a comprehensive tutorial for developers to create a rich video streaming service.
```

Comment:
@kazo0, @nickrandolph do you think "The TubePlayer App is a sample application that leverages Uno Platform **to recreate the core functionalities of YouTube**" isn't a bit much here? 

As it is just a search and stream video experience.
![image](https://github.com/unoplatform/Uno.Samples/assets/16295702/4ae4f08f-a77e-4099-a836-6a365ec36b7f)